### ISSUE TITLE
document-title rule doesn't appear to be working in Axe Linter

### ISSUE THREAD

**Issue by ndewfraedom** [dequelabs/axe-core#3713]
### Product

axe Linter

### Product Version

4.4.2

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
When not including a <title>, or when including an empty <title>, in a plain HTML document, I expect to see an error displayed, based on the [document-title](https://dequeuniversity.com/rules/axe/4.4/document-title) rule.

#### Actual
No errors displayed when either not including a <title> or including an empty <title> in a plain HTML document.

#### How to Reproduce
```
<html>
	<h1>this page has no title</h1>
</html>
```

```
<html>
	<title></title>
</html>
```

**Comment by ndewfraedom** [comment]
Apologies, I'm assuming the list of rules which applies to axe-linter https://marketplace.visualstudio.com/items?itemName=deque-systems.vscode-axe-linter is less than the full list that applies to axe-core https://dequeuniversity.com/rules/axe/4.4/. I will close this ticket!
